README.md

```diff
@@ -1,4 +1,4 @@
-# Astrobee Robot Software
+# Astrobee Robot Software v1
```

Comment:
awesome

---
**Sema Reaction:** :trophy: This code is awesome

Reply:
Always self-close tags that have no children. eslint: react/self-closing-comp

📄 [AirBnB](https://github.com/airbnb/javascript/tree/master/react)

Reply:
Always self-close tags that have no children. eslint: react/self-closing-comp

📄 [AirBnB](https://github.com/airbnb/javascript/tree/master/react)